src/loaders/common/loader_helper.py

```diff
@@ -69,11 +75,60 @@ def init_genome_atrri_doc(kbase_collection, load_version, genome_id):
     # The '_key' field for the document should be generated by applying a hash function to a combination of the
     # 'kbase_collection', 'load_version', and 'genome_id' fields.
     doc = {
-        names.FLD_ARANGO_KEY: md5_string(f"{kbase_collection}_{load_version}_{genome_id}"),
+        names.FLD_ARANGO_KEY: md5_string(
+            f"{kbase_collection}_{load_version}_{genome_id}"
+        ),
         names.FLD_COLLECTION_ID: kbase_collection,
         names.FLD_LOAD_VERSION: load_version,
         names.FLD_GENOME_ATTRIBS_KBASE_GENOME_ID: genome_id,  # Begin with the input genome_id, though it may be altered by the calling script.
-        names.FLD_GENOME_ATTRIBS_MATCHES_SELECTIONS: []  # for saving matches and selections
+        names.FLD_GENOME_ATTRIBS_MATCHES_SELECTIONS: [],  # for saving matches and selections
     }
 
     return doc
+
+
+def get_token(token_filename):
+    """
+    Get token if a filename is provided at home dir.
+    """
+    try:
+        home_dir = os.path.expanduser("~")
+        with open(os.path.join(home_dir, "." + token_filename), "r") as f:
+            token = f.readline().strip()
+    except FileNotFoundError as e:
+        print(e.errno)
+    return token
+
+
+def start_podman_service(uid):
+    """
+    Start podman service. Used by workspace_downloader.py script.
+    """
+    # try:
+    #     check_output(["pidof", "podman"])
+    # except CalledProcessError:
+    #     print("No running podmans servies are detected. Start one now!")
+
+    # TODO find out the right way to check if a podman service is running
+    proc = subprocess.Popen(["podman", "system", "service", "-t", "0"])
+    os.environ["DOCKER_HOST"] = DOCKER_HOST.format(uid)
+    return proc
+
+
+def is_upaInfo_complete(output_dir, upa):
```

Comment:
ok but according to PEP8, function and variable names should be lowercase.

Reply:
:+1: